### ISSUE TITLE
Add addresses in suburbs of Detroit, MI

### ISSUE THREAD

**Issue by nvkelso** [openaddresses/openaddresses#1995]
Status:
- [x] **Research sources** and propose best source (and it's data license)
  - [x] Wayne county
  - [x] Oakland county
  - [ ] Macomb county
- [ ] **Review proposed source**
- [ ] **Approve source** based on review comments
- [ ] **Add** to OpenAddresses

_NOTE: Proposed source notes should include basic column mapping to OpenAddress data model._

**Comment by justinelliotmeyers** [comment]
Contacted Wayne County requesting data 10/2/2016

**Comment by justinelliotmeyers** [comment]
Contacted Oakland County requesting data 10/2/2016

**Comment by justinelliotmeyers** [comment]
Contacted Macomb County requesting data 10/2/2016

**Comment by justinelliotmeyers** [comment]
Oakland County response:
Justin,
 
I received your request below, Oakland County GIS data can be purchased through the Oakland County One Stop Shop
One Stop Shop Business Center
Economic Development & Community Affairs Oakland County, Michigan
248.452.2058
 
Cost is $.01 per record and there are 491,901 records ($4,919.01).  OSS office charge of $30.  Total would be $4,949.01.
We also require a signed user agreement (see attached).
 
Let me know if you have any questions.
Thank you,
Janine
 
Janine Daily GISP, IT Business Analyst
Oakland County Department of Information Technology
1200 N. Telegraph Road | Building 49W | Pontiac, MI  48341
248.858.0123 direct  |  Fax 248.452.9128

**Comment by justinelliotmeyers** [comment]
As per Oakland County, might be a better idea to contact individual cities to see if they will share without such a large cost.

**Comment by nvkelso** [comment]
Yikes! Let's see if individual cities have better price options (hopefully
free).

**Comment by justinelliotmeyers** [comment]
Macomb County Update from Dan Durkee (dan.durkee@macombgov.org )
1st email response: Hi Justin, we do not have a distribute-able point address data set to provide you.  thanks for your interest

I emailed him back asking how I could succesfully achieve the task without the counties data.

2nd response: Hi Justin, you could ask each community's tax assessor(addressing authority) in Macomb County for a list of addresses to geocode. 
We work primarily with address ranges(address approximation) in street center-lines.

**Comment by dexterslu** [comment]
@justinelliotmeyers @nvkelso Oakland County rest service parcel points here:
https://gisservices.oakgov.com/arcgis/rest/services/Enterprise/EnterpriseOpenParcelDataMapService/MapServer/0 

Parcel polygons here:
https://gisservices.oakgov.com/arcgis/rest/services/BaseMap/EnterpriseCAMSBasemapMap/MapServer/132

**Comment by iandees** [comment]
Thanks @dexterslu. I got that in #2117. 👍

**Comment by dexterslu** [comment]
@iandees great! Also, here is Wayne County proper:
http://services1.arcgis.com/b6rkZNtCd6Mx2gvB/ArcGIS/rest/services/parcel_updates_11_2016/FeatureServer/0

**Comment by nvkelso** [comment]
Once we get the last open source in let's close this issue but create a new issue to track Macomb County.

**Comment by justinelliotmeyers** [comment]
http://arcgisserver.semcog.org/arcgis/rest/services/OpenData/Buildings/MapServer/0

is for: Detroit, and Livingston,Macomb,Monroe,Oakland,StClair,Washtenaw,Wayne counties

**Comment by justinelliotmeyers** [comment]
whoever pulls the data, let me know or just close out the ticket once it is done.  Thanks!
